cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,635 @@
+            if osp_security_group or osp_flavor:
+                view.instance_properties.wait_displayed()
+                view.instance_properties.edit_instance_properties.click()
+                view.instance_properties.select_security_group.wait_displayed()
+                if osp_security_group:
+                    view.instance_properties.select_security_group.fill(osp_security_group)
+                if osp_flavor:
+                    view.instance_properties.select_flavor.fill(osp_flavor)
+                view.instance_properties.save_properties.click()
+            view.next_btn.click()
+
+        if pre_playbook:
+            view.advanced.pre_playbook.wait_displayed("5s")
+            view.advanced.pre_playbook.fill(pre_playbook)
+            view.advanced.pre_checkbox.click()
+        if post_playbook:
+            view.advanced.post_playbook.wait_displayed("5s")
+            view.advanced.post_playbook.fill(post_playbook)
+            view.advanced.post_checkbox.click()
+        view.next_btn.click()
+
+        if start_migration:
+            view.schedule.run_migration.select("Start migration immediately")
+        view.schedule.create.click()
+        wait_for(
+            lambda: view.results.msg.is_displayed, timeout=60, message="Wait for Results view"
+        )
+
+        base_flash = "Migration Plan: '{}'".format(name)
+        if start_migration:
+            base_flash = "{} is in progress".format(base_flash)
+        else:
+            base_flash = "{} has been saved".format(base_flash)
+        assert view.results.msg.text == base_flash
+        view.results.close.click()
+        return self.instantiate(name)
+
+    def is_plan_started(self, name):
+        view = navigate_to(self, "All")
+        return wait_for(
+            func=view.progress_card.is_plan_started,
+            func_args=[name],
+            message="migration plan is starting, be patient please",
+            delay=5,
+            num_sec=150,
+            handle_exception=True
+        )
+
+
+@navigator.register(MigrationPlanCollection, "All")
+class All(CFMENavigateStep):
+    prerequisite = NavigateToAttribute("appliance.server", "LoggedIn")
+    VIEW = MigrationPlanView
+
+    def step(self):
+        if self.obj.appliance.version < "5.10":
+            self.prerequisite_view.navigation.select("Compute", "Migration")
+        else:
+            self.prerequisite_view.navigation.select("Compute", "Migration", "Migration Plans")
+
+
+@navigator.register(MigrationPlanCollection, "Add")
+class AddMigrationPlan(CFMENavigateStep):
+    VIEW = AddMigrationPlanView
+    prerequisite = NavigateToSibling("All")
+
+    def step(self):
+        self.prerequisite_view.create_migration_plan.click()
+
+
+@navigator.register(MigrationPlan, "NotStarted")
+class NotStartedPlans(CFMENavigateStep):
+    prerequisite = NavigateToAttribute("parent", "All")
+
+    VIEW = NotStartedPlansView
+
+    def step(self):
+        if self.obj.appliance.version < "5.10":
```

Comment:
We've set a breakpoint for 5.9, and this logic can be simplified to support 5.10+